handbook_reviewer_guidelines.md

```diff
@@ -1,25 +1,29 @@
-This document contains the guidelines for anyone reviewing content for the Operate First Community Handbook.
+# Handbook Reviewer Guidelines
```

Comment:
I think we're going to find that adopting sentence case for titles makes the whole handbook easier to read.

This means it would be:

# Handbook reviewer guidelines

Then the only thing with a proper Title Case caps is the name, "Operate First Community Handbook".

Reply:
Okay, that sounds good! I will update these